server/models/referralUpdate.ts

```diff
@@ -0,0 +1,6 @@
+import DraftReferral from './draftReferral'
+
+export interface ReferralDetailsFormUpdate {
```

Comment:
should this live in the models directory or alongside the form?

Reply:
In the future would we not use this for other referral details forms(enforceable days)? All of these would live in slightly different places.